File: Number_Conversion/Python/Decimal_To_Octal.py
def decToOctal(num):       
    octal = [0] * 100
    
    i = 0
    while (num != 0):   
        octal[i] = num % 8 
        num = int(num / 8)
        i += 1  
    
    for j in range(i - 1, -1, -1): 
        print(octal[j], end = "")
 
num = int(input())  
decToOctal(num)

'''
Input:
20
Output:
24
'''

File: Number_Conversion/C++/Decimal_To_Octal.cpp
#include<iostream> 
using namespace std;   
 
int main() 
{ 
    int num;
    cin>>num; 
  
    // To store octal number 
    int octal[100]; 
  
    int i = 0, j; 
    while (num != 0) 
    {   
        octal[i] = num % 8; 
        num = num / 8; 
        i++; 
    } 
  
    for (j = i - 1; j >= 0; j--) 
        cout<<octal[j];
  
    return 0; 
} 

/*
Input:
20
Output:
24
*/
